### PR TITLE
Add SLE_12_SP4 to openstack-cleanvm heat template

### PR DESCRIPTION
**Pull request by JanZerebecki**
Also fix setting variable OSHEAD.

### CODE REVIEW

File: jenkins/ci.opensuse.org/heat-templates/deploy-cleanvm.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ IS_HEAT=_NOTHEAT
 
 if [ "$IS_HEAT" = HEAT ]; then
   # When run by Heat template, these will be substituted with the template's parameters
-  TESTHEAD=_TESTHEAD
+  OSHEAD=_TESTHEAD
```
Comment (toabctl) [inline jenkins/ci.opensuse.org/heat-templates/deploy-cleanvm.sh]: `OSHEAD` is used in jenkins/ci.opensuse.org/openstack-cleanvm.yaml . Also what's wrong with `OSHEAD`?

Comment (JanZerebecki) [reply jenkins/ci.opensuse.org/heat-templates/deploy-cleanvm.sh]: Also OSHEAD is used in qa_openstack.sh which is called below and why we set it here. Nothing is wrong with OSHEAD, it is just not named TESTHEAD, which is what this change corrects.

File: jenkins/ci.opensuse.org/heat-templates/deploy-cleanvm.sh [hunk 2]
```diff
@@ -36,7 +36,11 @@ fi
 zypper='zypper --non-interactive'
 
 case "$VERSION_ID" in
-  12.3)
+  "12.4")
+    $zypper ar 'http://smt-internal.opensuse.org/repo/$RCE/SUSE/Products/SLE-SERVER/12-SP3/x86_64/product/' SLE12-SP3-Pool
+    $zypper ar -f 'http://smt-internal.opensuse.org/repo/$RCE/SUSE/Updates/SLE-SERVER/12-SP3/x86_64/update/' SLES12-SP3-Updates
```
Comment (dirkmueller) [inline jenkins/ci.opensuse.org/heat-templates/deploy-cleanvm.sh]: Copy and paste error

Comment (JanZerebecki) [reply jenkins/ci.opensuse.org/heat-templates/deploy-cleanvm.sh]: Fixed

**Pull request merged**